pylint_django/checkers/forms.py

```diff
@@ -41,7 +41,8 @@ def visit_classdef(self, node):
             return
 
         for child in meta.get_children():
-            if not isinstance(child, Assign):
+            if (not isinstance(child, Assign)
+                    or not isinstance(child.targets[0], AssignName)):
```

Comment:
IMO you should copy the example code from #284 into one of the existing tests. `func_noerror_forms_py33.py` looks like a good candidate.

Reply:
Hi! I took a look at the test files and got a bit lost, then couldn't find the time to read all the files in detail, so thank you for pointing me out to the best candidate. I'll give it a try.